### ISSUE TITLE
io.read_nwb issues with new nwb release (now pinned to pynwb<1.4)

### ISSUE THREAD

**Issue by chriski777** [MouseLand/suite2p#509]
nwb 1.4.0 was released a couple of hours ago and io.save_nwb is broken. Test shows the following
```
suite2p/io/nwb.py:80: in read_nwb

    F = np.array(nwbfile.processing['ophys']['Fluorescence']['roi_response_series'].data)
>               raise KeyError(msg)

E               KeyError: "'roi_response_series' not found in Fluorescence 'Fluorescence'."

.eggs/hdmf-2.1.0-py3.7.egg/hdmf/container.py:663: KeyError
```

**Comment by chriski777** [comment]
As of now, we've pinned pynwb to be < 1.4. Once appropriate changes in `io.nwb` to work with pynwb 1.4.0 are made, this pin can be removed from setup.py.

**Comment by carsen-stringer** [comment]
@chriski777 or @nickdelgrosso can you post a report on this issue on [pynwb](https://github.com/NeurodataWithoutBorders/pynwb/issues) please? I'm not sure I fully understand the problem -- what part of saving fails so that reading fails as you've stated?

**Comment by chriski777** [comment]
Sure, can do. I don't understand exactly what's going on either as I didn't look too much into it. It seems saving is fine and this is more on the reading end. I'll have to look further into it but I think they changed how you access roi_response_data in `nwbfile.processing['ophys']['Fluorescence']`

**Comment by chriski777** [comment]
issue opened [here](https://github.com/NeurodataWithoutBorders/pynwb/issues/1283)

**Comment by chriski777** [comment]
Solved by #515 and unpinned in #516.
